tests/libres_tests/res/enkf/test_enkf.py

```diff
@@ -74,6 +77,30 @@ def test_bootstrap(self):
             main = EnKFMain(res_config)
             self.assertTrue(main, "Load failed")
 
+    @tmpdir()
```

Comment:
Don't create new old-style tests. Create a new pytest test at the bottom of the file.

Reply:
Perhaps a new testfile in `libres_tests/res/config` ?  What would be a good template for a modern test?

Reply:
Have a look at `ert_tests/ert3` for some good new style tests.

Reply:
As you can see here: https://github.com/equinor/ecl/blob/3d6d17e1470419fc5af0c891560fb2211b2eda0d/python/tests/ecl_tests/test_ecl_sum.py
A file can both contain tests in a class and function tests (both the members of `EclSumTest` and `test_write_repr` will be run).